packages/http-smoke-testing/src/Test/TestController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\HttpSmokeTesting\Test;
+
+use Shopsys\HttpSmokeTesting\Annotation\DataSet;
+use Shopsys\HttpSmokeTesting\Annotation\Parameter;
+use Shopsys\HttpSmokeTesting\Annotation\Skipped;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class TestController
+{
+    /**
+     * @param string $name
+     * @return \Symfony\Component\HttpFoundation\Response
+     * @Route("/hello/{name}")
+     *
+     * @DataSet(parameters={
+     *     @Parameter("name", value="Batman")
```

Comment:
I'm not sure what am I doing wrong, but when I use annotation like this in CartController the test fail on undefined `$cartItemId`(I'm not defining the `$request` but only `$cartItemId`)...
 https://github.com/shopsys/shopsys/blob/466f10c7bf4a11abb4ff423b364ed1ca1f569ba8/project-base/src/Shopsys/ShopBundle/Controller/Front/CartController.php#L309

When I changed the annotatnion like the one below, everything went fine
```suggestion
     *     @Parameter(name="name", value="Batman")
```

Do I missing something? Maybe it is expected behavior, but it suprised me.

Reply:
What version of `doctrine/annotations` is in use?

Reply:
We have defined `"doctrine/migrations": "^1.6.0",` in appplication. So maybe that's it.

Reply:
That is a different library, though. I just saw you are using `"doctrine/annotations": "^1.6.0"`. I will test with this library. Maybe this issue will also appear.

Reply:
Ok, I was able to address this in the tests. You will need the `name` attribute in the `@Parameter` annotation.

I also marked all properties in the `@Parameter` annotation as required.

Reply:
So can we mark this as resolved?

Reply:
Oh, sorry for that. I just copied wrong line 🤦‍♂ . I will check it once more in the evening. But I'm a bit woried we will need to udpdate `doctrine/annotations` in this repository. In that case we will need to release this in next major 😞

Reply:
I was writing faster than reviewing. I'm resolving this one 👍